### ISSUE TITLE
Debian/Ubuntu installation is not working

### ISSUE THREAD

**Issue by sratatata** [mwrlabs/drozer#217]
I've tried to use your instruction: 

> apt-get install drozer

but drozer package is not existing, i've also tried to search for it: 

> apt-cache search drozer 

same result. 

Do we need to use some specific ppa?

**Comment by HenryHoggard** [comment]
Thanks for reporting this, will look into this now.

**Comment by ArturSpirin** [comment]
Any updates on this? Links on the website do not work neither for download.

**Comment by SleepProgger** [comment]
The download links on the mwr site 404.
I managed to install drozer on debian jessie by building this master branch.

I had to copy the meta.py from the src folder to the "root" folder (where setup.py is located) and change the import of it in the setup.py script from "from drozer import meta".to "import meta" and also  needed to "apt-get install python-cryptography" before.

Most stuff works but i am no having the "drozer could not find or compile a required extension library" error. There is no JDK6 anymore in the repos at all and i failed (due to dependency problems) to build and install it from source. I am looking forward to a fix.

For now i guess using an old debian vm should work to build the apks via "make apks", but thats no fun.
